lib/TransferManager.cs

```diff
@@ -230,38 +232,44 @@ public static Task UploadAsync(Stream sourceStream, CloudFile destFile)
         /// Upload a list of files to Azure Blob Storage.
         /// </summary>
         /// <param name="transferItems">List of files to be uploaded.</param>
-        /// <returns>A <see cref="Task{T}"/> object of type <see cref="TransferStatus"/> that represents the asynchronous operation.</returns>
-        public static Task<TransferStatus> UploadAsync(IEnumerable<TransferItem> transferItems)
-        {
-            return UploadAsync(transferItems, null, null);
-        }
-
-        /// <summary>
-        /// Upload a list of files to Azure Blob Storage.
-        /// </summary>
-        /// <param name="transferItems">List of files to be uploaded.</param>
-        /// <param name="options">Object that specifies additional options for the operation.</param>
-        /// <param name="context">Object that represents the context for the current operation.</param>
+        /// <param name="destBlobDir">Destination root to be uploaded to.</param>
+        /// <param name="cancellationToken">Cancellation</param>
         /// <returns>A <see cref="Task{T}"/> object of type <see cref="TransferStatus"/> that represents the asynchronous operation.</returns>
         public static Task<TransferStatus> UploadAsync(IEnumerable<TransferItem> transferItems,
-            UploadOptions options, DirectoryTransferContext context)
+            CloudBlobDirectory destBlobDir, CancellationToken cancellationToken)
```

Comment:
```suggestion
            CloudBlobDirectory rootDestBlobDir, CancellationToken cancellationToken)
```
We need to emphasize that this is not a destination directory but a root folder where files are uploaded (using relative paths).